### PR TITLE
dForms 14513

### PR DESCRIPTION
**Pull request by pranay-tippa**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--aem-boilerplate-forms--adobe-rnd.hlx.live/
- After: https://forms-14513--aem-boilerplate-forms--adobe-rnd.hlx.page/

### CODE REVIEW

File: test/e2e/utils.js [hunk 1]
```diff
@@ -10,3 +10,36 @@ function executeGitCommand(command) {
 export function getCurrentBranch() {
   return executeGitCommand('git rev-parse --abbrev-ref HEAD');
 }
+// eslint-disable-next-line max-len
+export async function fillField(page, coreComponent, textInput, emailInput, numberInput, dropDown, FilePath, dataInput) {
```
Comment (deepprakash345) [inline test/e2e/utils.js]: Refactor this method, 8 arguments is too many

Comment (deepprakash345) [reply test/e2e/utils.js]: what does coreComponent argument represent here, from the code it seems like it represents label of the component.
In that case please term it accordingly, its difficult understand the code by reading.

File: test/e2e/utils.js [hunk 2]
```diff
@@ -10,3 +10,36 @@ function executeGitCommand(command) {
...
+      break;
+    case 'File Attachment':
+      await page.getByLabel(coreComponent).setInputFiles(FilePath);
+      break;
+    case 'Date Input':
+      await page.getByLabel(coreComponent).focus();
+      await page.getByLabel(coreComponent).fill(dataInput);
+      break;
+    default:
+      break;
```
Comment (deepprakash345) [inline test/e2e/utils.js:40]: can you add a log in case default block is reached. If this scenario shouldn't happen then can you add it as an error log.

**Review by deepprakash345 — DISMISSED** [review]
inline comments

File: test/e2e/utils.js (hunk 2 above)
Comment (pranay-tippa) [reply test/e2e/utils.js:40]: Added error log in default block in both switch cases

File: test/e2e/utils.js (hunk 1 above)
Comment (pranay-tippa) [reply test/e2e/utils.js]: Hi Deep, changed coreCompoent argument to readable name.

**Review by ravise5 — APPROVED** [review]
LGTM

**Pull request merged**